File: src/main/java/hacker/solution/StaticBlock.java
package hacker.solution;

import java.util.Scanner;

public class StaticBlock {
    public static void main(String[] args) {
        if(flag) {
            int area = B * H;
            System.out.print(area);
        }
    }

    public static int B = 0;
    public static int H = 0;
    public static boolean flag = true;

    static {
        Scanner sc = new Scanner(System.in);
        try {
            B = sc.nextInt();
            H = sc.nextInt();
            if (B < 1 || H < 1) {
                flag = false;
                throw new Exception("Breadth and height must be positive");
            }
        } catch (Exception e) {
            System.out.println("java.lang.Exception: " + e.getMessage());
        }

    }
}

File: src/main/kotlin/hacker/solutionk/StaticBlock.kt
package hacker.solutionk

import java.util.*

object StaticBlock {

    var B = 0
    var H = 0
    var flag = true
    @JvmStatic
    fun main(args: Array<String>) {
        if (flag) {
            val area = B * H
            print(area)
        }
    }

    init {
        val sc = Scanner(System.`in`)
        try {
            B = sc.nextInt()
            H = sc.nextInt()
            if (B < 1 || H < 1) {
                flag = false
                throw Exception("Breadth and height must be positive")
            }
        } catch (e: Exception) {
            println("java.lang.Exception: " + e.message)
        }

    }
}